echion/stacks.h

```diff
@@ -103,12 +103,39 @@ unwind_frame(PyObject *frame_addr, FrameStack &stack)
         if (seen_frames.find(current_frame_addr) != seen_frames.end())
             break;
 
+#if PY_VERSION_HEX >= 0x030d0000
+        // From Python versions 3.13, f_executable can have objects other than
+        // code objects for an internal frame. We need to skip some frames if
+        // its f_executable is not code as suggested here:
+        // https://github.com/python/cpython/issues/100987#issuecomment-1485556487
+        _PyInterpreterFrame iframe;
+        PyObject f_executable;
```

Comment:
I think we should propagate to calls to `Frame::read` below to avoid extra `copy_type` call but that requires some more editing code...